### PR TITLE
frontend/fix: Updated configs and handled trips with same destination…

### PR DESCRIPTION
**Pull request by aryan-13**
… location

## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-customer/src/Flow.purs [hunk 1]
```diff
@@ -2528,10 +2532,17 @@ fetchAndModifyLocationLists savedLocationResp = do
...
               , suggestionsData{suggestionsMap = suggestionsMap}
-              , tripSuggestions = sortedTripList
+              , tripSuggestions = removeDuplicateTrips sortedTripList 
               }
             })
     where
+
+      removeDuplicateTrips :: Array Trip -> Array Trip
+      removeDuplicateTrips trips = 
+        let grouped = groupBy (\a b -> (getGeoHash a.destLat a.destLong 9) == (getGeoHash b.destLat b.destLong 9)) trips
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/src/Flow.purs:2542]: @aryan-13 instead of a, b can you give better variable name

Comment (kavya-shree-s) [reply Frontend/ui-customer/src/Flow.purs:2542]: also why Geohash precision is 9, on what basis this is decided ?

Comment (kavya-shree-s) [reply Frontend/ui-customer/src/Flow.purs:2542]: Can we make geohash precision configurable

Comment (aryan-13) [reply Frontend/ui-customer/src/Flow.purs:2542]: So as to remove the trips with destinations within 4.8x4.8m block.

**Review by kavya-shree-s — APPROVED** [review]

**Pull request merged**